database/models.go

```diff
@@ -22,47 +22,129 @@ import (
 	"github.com/coreos/clair/pkg/pagination"
 )
 
-// Processors are extentions to scan a layer's content.
-type Processors struct {
-	Listers   []string
-	Detectors []string
-}
-
 // Ancestry is a manifest that keeps all layers in an image in order.
 type Ancestry struct {
+	// Name is expected to be the sha-256 digest of the manifest for a
```

Comment:
Name is a globally unique value for a set of layers. This is often the sha256 digest of an OCI/Docker manifest.